man/generateRdFiles.Rd

```diff
@@ -0,0 +1,89 @@
+% Generated by roxygen2: do not edit by hand
+% Please edit documentation in R/PythonPkgWrapperUtils.R
+\name{generateRdFiles}
+\alias{generateRdFiles}
+\title{Generate .Rd files for Python classes and functions}
+\usage{
+generateRdFiles(srcRootDir, pyPkg, module, modifyFunctions = NULL,
+  modifyClasses = NULL, functionPrefix = NULL, keepContent = FALSE,
+  templateDir = NULL)
+}
+\arguments{
+\item{srcRootDir}{The directory of the R package}
```

Comment:
The root directory under which another directory, 'auto-man/' is created to hold the output, Rd files.

Somewhere we need to explain the rationale for writing into 'auto-man/' rather than directly to 'man/'.